modules/admin_manual/pages/configuration/user/user_auth_ldap.adoc

```diff
@@ -249,15 +252,27 @@ user upon login.
 *Examples:*
 
 * Only username:
-
++
+[source,ldap]
 ----
-(&(objectClass=inetOrgPerson)(memberOf=cn=owncloudusers,ou=groups,dc=example,dc=com)(uid=%uid)
+(&
+  (objectClass=inetOrgPerson)
+  (memberOf=cn=owncloudusers,ou=groups,dc=example,dc=com)
+  (uid=%uid)
+)
 ----
 
 * Username or email address:
-
++
+[source]
 ----
-((&(objectClass=inetOrgPerson)(memberOf=cn=owncloudusers,ou=groups,dc=example,dc=com)(|(uid=%uid)(mail=%uid)))
+(
+  (&
+    (objectClass=inetOrgPerson)
+    (memberOf=cn=owncloudusers,ou=groups,dc=example,dc=com)
+    (|(uid=%uid)(mail=%uid)
+  )
+)
```

Comment:
Shouldn't this be `(|(uid=%uid)(mail=%uid)` --> `(|(uid=%uid)(mail=%uid))` ?
Counting the open/closing brackets, the closing one at the end is imho missing.
You could improve readability (if this is possible) by
```
(|
  (uid=%uid)
  (mail=%uid)
)
```

Reply:
I've been doing some investigating of LDIF validation tools and have found one, but I don't know how practical it would be. Will keep you posted.

Reply:
I've extracted the main LDIF examples into example files.